exporter/awsemfexporter/emf_exporter.go

```diff
@@ -45,11 +44,10 @@ const (
 )
 
 type emfExporter struct {
-	// Each (log group, log stream) keeps a separate pusher because of each (log group, log stream) requires separate stream token.
-	groupStreamToPusherMap map[string]map[string]cwlogs.Pusher
-	svcStructuredLog       *cwlogs.Client
-	config                 component.Config
-	logger                 *zap.Logger
+	pusherMap        map[cwlogs.PusherKey]cwlogs.Pusher
```

Comment:
Added some clean up here. Map is unquie by the log group / stream which we can turn into a key. Instead of having a map of map.